package.json

```diff
@@ -185,6 +184,9 @@
     "secure-json-parse": "^2.0.0",
     "tiny-lru": "^7.0.0"
   },
+  "peerDependencies": {
+    "@types/node": "^14.10.3"
+  },
```

Comment:
I'm -1 to this specific change. `peerDependency` semantics is different between package managers.

Reply:
and opinion on shipping @types/node as a dep?

Reply:
Many people frown upon types being shipped as deps, as that means they will be retained on runtime.

Reply:
Yeah I agree. I wonder what the best way to indicate users need to download @types/node is 🤔 (the official ts docs say to ship them as a dep)

Reply:
I'm strongly -1 to having them as a dep as well.

Reply:
I am -1 for peerDependencies too. I consider this an unresolved issue of the "type system dependency". I don't like adding them to dependencies either, though. :/